controller_manager:
  ros__parameters:
    update_rate: 100

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController

arm_controller:
  ros__parameters:
    joints:
      - joint1
      - joint2
    command_interfaces: 
      - position
    state_interfaces:
      - position
      - velocity
    gains:
      joint1:
        p: 5.0
        i: 0.0
        d: 0.5
      joint2:
        p: 5.0
        i: 0.0
        d: 0.5
